uber/site_sections/preregistration.py

```diff
@@ -93,9 +93,10 @@ def index(self, session, message=''):
         else:
             charge = Charge(listify(Charge.unpaid_preregs.values()))
             for attendee in charge.attendees:
-                if attendee.promo_code and attendee.promo_code.group:
-                    attendee.group_name = session.query(PromoCode).filter_by(
-                        code=attendee.promo_code.code).first().group.name
+                if attendee.promo_code:
```

Comment:
The emergency fix in https://github.com/magfest/ubersystem/pull/3445 introduced a bug where the promo code group's name would never actually display on the index page. This fixes that.